src/astro/files/locations/s3.py

```diff
@@ -0,0 +1,44 @@
+import os
+from typing import Dict, List, Tuple
+from urllib.parse import urlparse, urlunparse
+
+from astro.constants import FileLocation
+from astro.files.locations.base import Location
+from astro.utils.dependencies import BotoSession, s3
+
+
+class S3(Location):
+    """Handler S3 object store operations"""
+
+    location_type = FileLocation.S3
+
+    @staticmethod
+    def _parse_s3_env_var() -> Tuple[str, str]:
+        """Return S3 ID/KEY pair from environment vars"""
+        return os.environ["AWS_ACCESS_KEY_ID"], os.environ["AWS_SECRET_ACCESS_KEY"]
+
+    def get_transport_params(self) -> Dict:
+        """Structure s3fs credentials from Airflow connection.
+        s3fs enables pandas to write to s3
+        """
+        if self.conn_id:
+            session = s3.S3Hook(aws_conn_id=self.conn_id).get_session()
+        else:
+            key, secret = self._parse_s3_env_var()
+            session = BotoSession(
+                aws_access_key_id=key,
+                aws_secret_access_key=secret,
+            )
```

Comment:
Should we do same as L39.

```suggestion
        hook = s3.S3Hook(aws_conn_id=self.conn_id) if self.conn_id else s3.S3Hook()
        session = hook.get_session()
```

Reply:
That way we can fallback to default conn_id and remove `_parse_s3_env_var(` method, wdyt?

Reply:
Done.